codespell_lib/data/dictionary.txt

```diff
@@ -25201,13 +25212,15 @@ liasing->liaising
 liasion->liaison
 liason->liaison
 liasons->liaisons
+libararies->libraries
 libarary->library
 libaries->libraries
 libary->library
 libell->libel
 liberaries->libraries
 liberary->library
 liberoffice->libreoffice
+liberries->libraries
 liberry->library
```

Comment:
While I agree library is the most probable fix, I wonder whether this could `liberty` or even `bilberry`.

Reply:
`liberty`: added
`bilberry`: not added, because this word is rare